Apply title rules to test.step

Albinutte opened the pull request:
Updated `valid-title` rule to apply to `test.step` commands.

mskelton commented on src/rules/valid-title.test.ts:
```diff
@@ -52,10 +52,10 @@ runRuleTester('valid-title', rule, {
     },
     {
-      code: 'test(`that the value is set properly`, function () {})',
+      code: 'test.step(`that the value is set properly`, function () {})',
```
Let's leave the existing test and just add new ones. Don't like changing existing tests ever.

Albinutte commented:
@mskelton Thanks for the review! I addressed your comment (it was only about the first test case, right?) and pushed a tiny update to rule description.
